docusaurus/docusaurus.config.js

```diff
@@ -22,14 +22,6 @@ const config = {
   },
   markdown: {
     mermaid: true,
-    preprocessor: ({filePath, fileContent}) => {
-      return fileContent
-        .replace(/\{\{product_name_sm_oss\}\}/g, 'Core')
```

Comment:
It seems there are a few places where this was not replaced:
https://github.com/airbytehq/airbyte/blob/f2057e30cdff43631750aab9666eadc6d8e19c0b/docusaurus/platform_versioned_docs/version-2.0/readme.md#L22-L23

<img width="483" height="436" alt="Image" src="https://github.com/user-attachments/assets/c5e828f9-c786-4695-9992-23bdd5ba2420" />